compose/ui/ui/src/jsNativeMain/kotlin/androidx/compose/ui/native/ComposeLayer.jsNative.kt

```diff
@@ -51,6 +52,16 @@ internal class ComposeLayer(
     init {
         layer.renderDelegate = object : SkikoRenderDelegate {
             override fun onRender(canvas: Canvas, width: Int, height: Int, nanoTime: Long) {
+
+                // Skiko subscribes to resize events itself, so make sure that
+                // the size is up-to-date during rendering.
+                val currentSceneSize = scene.size
+                if (currentSceneSize == null ||
+                    currentSceneSize.width != width ||
+                    currentSceneSize.height != height) {
+                    setSize(width, height)
```

Comment:
Calling `setSize` inside `onRender` is questionable. We should figure out the best way to avoid races with other calls of `setSize` before merging.